src/Components/MovieListCard/MovieListCard.tsx

```diff
@@ -22,25 +25,45 @@ const dropdownItems = [
 ];
 
 export function MoviesListCard({ movie }: MoviesListCardProps) {
+  const { title, poster_path, release_date, genres } = movie;
   const [isContextMenuOpen, setIsContextMenuOpen] = useState(false);
+  const [movieToDelete, setMovieToDelete] = useState<Movie | null>(null);
+  const [movieToEdit, setMovieToEdit] = useState<Movie | null>(null);
 
-  function handleEdit() {
+  function handleEditClicked() {
     setIsContextMenuOpen(false);
-
-    console.log('edit selected');
+    setMovieToEdit(movie);
   }
 
-  function handleDelete() {
+  function handleDeleteClicked() {
     setIsContextMenuOpen(false);
-
-    console.log('delete selected');
+    setMovieToDelete(movie);
   }
 
   function handleMovieSelect() {
     console.log('movie selected');
   }
 
-  const { title, poster_path, release_date, genres } = movie;
+  function handleMovieDelete() {
+    setMovieToDelete(null);
+    alert('Delete movie: ' + movieToDelete!.id);
+  }
+
+  function handleMovieEdit(formValue: Partial<Movie>) {
+    console.log(formValue);
+  }
+
+  const deleteMovieModal = movieToDelete ? (
```

Comment:
Just note. For working with modal windows useDisclosure hook could be useful.
As example you can see https://chakra-ui.com/docs/styled-system/utility-hooks/use-disclosure

Reply:
Ok. Thank you for advice. I will know for future.